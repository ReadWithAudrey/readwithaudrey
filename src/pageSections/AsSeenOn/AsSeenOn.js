import React from 'react'
import StyledAsSeenOn from './AsSeenOn.style'
import PropTypes from 'prop-types'
import gadgette from '../../images/gadgettelogo.png'
import lifehacker from '../../images/lifehackerlogo.png'
import irishexaminer from '../../images/irishexaminerlogo.png'
import thriveglobal from '../../images/thrivegloballogo.jpeg'

const AsSeenOn = () => (
  <StyledAsSeenOn>
    <h2 className="f1 pink tc fw7">As seen on:</h2>
    <div id="logos" className="w-100 flex items-center justify-center flex-column flex-row-ns">
      <a href="https://www.gadgette.com/2018/06/17/read-with-audrey/" target="_blank" rel="noopener noreferrer"><img src={gadgette} alt="Gadgette logo" className="logo ma4" /></a>
      <a href="https://lifehacker.com/why-you-should-read-a-book-with-a-stranger-1827717296" target="_blank" rel="noopener noreferrer"><img src={lifehacker} alt="Lifehacker logo" className="logo ma4" /></a>
      <a href="https://www.irishexaminer.com/breakingnews/technow/meet-audrey-the-online-book-app-where-users-read-aloud-to-one-another-854650.html" target="_blank" rel="noopener noreferrer"><img src={irishexaminer} alt="Irish Examiner logo" className="logo ma4" /></a>
      <a href="https://www.thriveglobal.com/stories/36262-audrey" target="_blank" rel="noopener noreferrer"><img src={thriveglobal} alt="Thrive Global logo" className="logo ma4" /></a>
    </div>
  </StyledAsSeenOn>
)

AsSeenOn.propTypes = {
  siteTitle: PropTypes.string,
}

export default AsSeenOn
