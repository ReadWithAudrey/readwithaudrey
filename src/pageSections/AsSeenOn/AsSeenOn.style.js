import styled from 'styled-components'

const StyledAsSeenOn = styled.div.attrs({
  className:
    'flex flex-column items-center justify-between ph4 ph6-ns pv4 page-padding montserrat w-100',
})`
  background-color: var(--light-green);
`
export default StyledAsSeenOn
