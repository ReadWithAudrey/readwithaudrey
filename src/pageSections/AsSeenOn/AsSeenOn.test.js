import React from 'react'
import renderer from 'react-test-renderer'
import AsSeenOn from './AsSeenOn'

describe('AsSeenOn component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<AsSeenOn />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
