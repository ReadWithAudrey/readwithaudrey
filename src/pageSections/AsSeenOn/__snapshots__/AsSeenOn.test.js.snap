// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AsSeenOn component snapshot test renders correctly 1`] = `
<div
  className="sc-bdVaJa flex flex-column items-center justify-between ph4 ph6-ns pv4 page-padding montserrat w-100 jfBJLb"
>
  <h2
    className="f2 tc fw7"
    style={
      Object {
        "color": "#fd8181",
      }
    }
  >
    As seen on:
  </h2>
  <div
    className="w-100 flex items-center justify-center flex-column flex-row-l"
    id="logos"
  >
    <a
      href="https://www.gadgette.com/2018/06/17/read-with-audrey/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Gadgette logo"
        className="logo ma4"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://lifehacker.com/why-you-should-read-a-book-with-a-stranger-1827717296"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Lifehacker logo"
        className="logo ma4"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://www.irishexaminer.com/breakingnews/technow/meet-audrey-the-online-book-app-where-users-read-aloud-to-one-another-854650.html"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Irish Examiner logo"
        className="logo ma4"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://www.thriveglobal.com/stories/36262-audrey"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Thrive Global logo"
        className="logo ma4"
        src="test-file-stub"
      />
    </a>
  </div>
</div>
`;
